modules/cli/src/main/java/org/apache/ignite/cli/core/exception/handler/UnknownCommandExceptionHandler.java

```diff
@@ -28,8 +28,9 @@
 public class UnknownCommandExceptionHandler implements ExceptionHandler<UnknownCommandException> {
 
     @Override
-    public void handle(ExceptionWriter err, UnknownCommandException e) {
+    public int handle(ExceptionWriter err, UnknownCommandException e) {
         err.write(e.getMessage());
+        return 1;
```

Comment:
Are we shure that it is not 2?

Reply:
This is only for REPL mode so this it irrelevant, I'll change to 2 and add a comment.